### PR TITLE
Rename debtors view

### PR DESCRIPTION
**Pull request by Douglasokolaa**
# Description

Corrected debtors views name

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/debtor/create.blade.php [hunk 1]
```diff
@@ -1,181 +1,228 @@
...
+                                                <div class="col-md-6">
+                                                    <div class="form-group">
+                                                        <label for="firstname">First Name</label>
+                                                        <div class="input-group">
+                                                            <div class="input-group-prepend">
+                                                                <span class="input-group-text">
+                                                                    <i class="uil uil-atm-card"></i>
+                                                                </span>
+                                                            </div>
+                                                            <input type="email" class="form-control" id="firstname" aria-describedby="emailHelp" placeholder="Enter First Name">
```
Comment (Jeremiahiro) [inline resources/views/backend/debtor/create.blade.php:31]: input type of email has enter first name as placeholder

**Comment by Jeremiahiro** [comment]
please go through your work

File: resources/views/backend/debtor/create.blade.php (hunk 1 above)
Comment (Douglasokolaa) [reply resources/views/backend/debtor/create.blade.php:31]: Hello @Jeremiahiro  didn't modify the content just renamed the file

File: resources/views/backend/debtor/create.blade.php [hunk 2]
```diff
@@ -1,181 +1,228 @@
...
+                                                    </div>
+                                                </div>
+                                            </div>
+                                            <div class="form-group">
+                                                <label for="exampleInputEmail1">Email address</label>
+                                                <div class="input-group">
+                                                    <div class="input-group-prepend">
                                                         <span class="input-group-text">
-                                                            <i class="icon-dual" data-feather="lock"></i>
+                                                            @
```
Comment (eni4sure) [inline resources/views/backend/debtor/create.blade.php:54]: Please check this out @Douglasokolaa 
Shouldn't this be CSFR  ??

File: resources/views/backend/debtor/create.blade.php [hunk 3]
```diff
@@ -1,181 +1,228 @@
...
                                                         </span>
+                                                    </div>
+                                                    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter Email">
```
Comment (eni4sure) [inline resources/views/backend/debtor/create.blade.php:57]: Also, you can get rid of these id="exampleInput" ... and the aria-...
They are not needed

File: resources/views/backend/debtor/create.blade.php (hunk 1 above)
Comment (eni4sure) [reply resources/views/backend/debtor/create.blade.php:31]: Please modify the content @Douglasokolaa

**Pull request merged**